db/seeds.rb

```diff
@@ -14,6 +14,7 @@
           email: "test@test.com",
           password: "foobar99",
           password_confirmation: "foobar99",
+          state: 'NY'
```

Comment:
I upgraded postgres and had to start over, this was missing from the seed file, totally unrelated to what's trying to be done here.